src/fides/api/schemas/privacy_experience.py

```diff
@@ -56,56 +86,45 @@ def validate_regions(
 
     @root_validator
     def validate_attributes(cls, values: Dict[str, Any]) -> Dict[str, Any]:
-        """Validate minimum set of required fields exist given the type of component and/or delivery_mechanism"""
+        """Validate minimum set of required fields exist given the type of component"""
         component: Optional[ComponentType] = values.get("component")
-        delivery_mechanism: Optional[DeliveryMechanism] = values.get(
-            "delivery_mechanism"
-        )
-
-        if delivery_mechanism == DeliveryMechanism.link and not values.get(
-            "link_label"
-        ):
-            raise ValueError(
-                "Link label required when the delivery mechanism is of type link."
-            )
 
         if component == ComponentType.overlay:
-            if delivery_mechanism == DeliveryMechanism.banner:
-                required_banner_fields = [
-                    "acknowledgement_button_label",
-                    "banner_title",
-                    "confirmation_button_label",
-                    "reject_button_label",
-                ]
-                for field in required_banner_fields:
-                    if not values.get(field):
-                        raise ValueError(
-                            f"The following fields are required when defining a banner: {required_banner_fields}."
-                        )
-
-        if component == ComponentType.privacy_center:
-            if delivery_mechanism != DeliveryMechanism.link:
-                raise ValueError(
-                    "Privacy center experiences can only be delivered via a link."
-                )
+            required_overlay_fields = [
+                "banner_enabled",
+                "privacy_preferences_link_label",
+            ]
+            for field in required_overlay_fields:
+                if not values.get(field):
+                    raise ValueError(
+                        f"The following additional fields are required when defining an overlay: {human_friendly_list(required_overlay_fields)}."
+                    )
 
         return values
 
 
 class ExperienceConfigUpdate(ExperienceConfigSchema):
     """
-    Updating ExperienceConfig - requires regions to be patched specifically
+    Updating ExperienceConfig. Note that component cannot be updated once its created
     """
 
-    regions: List[PrivacyNoticeRegion]
+    regions: Optional[List[PrivacyNoticeRegion]] = Field(
+        default=None,
+        description="If None, no edits will be made to regions.  If an empty list, all regions will be removed.",
+    )
```

Comment:
As mentioned elsewhere, regions are no longer required in the request to update the ExperienceConfig. If you just want to update the copy and not the regions, don't include the regions key in the request at all.